packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/Javagen.java

```diff
@@ -254,7 +254,7 @@ protected JavaPackage writeToTemplates(CodeModel codeModel, Client client, JavaS
             javaPackage.addPackageInfo(packageInfo.getPackage(), "package-info", packageInfo);
         }
 
-        if (settings.isDataPlaneClient()) {
+        if (settings.isDataPlaneClient() || settings.isUnbranded()) {
```

Comment:
This is a fix for a regression that POM.xml not generated in unbranded.

Reply:
Emm, do I need to add `isAzureV2` as well? I assume v2 SDK also need a POM?

Currently POM is not generated for vanilla. Unsure whether we have such in v2.